src/cf-base.less

```diff
@@ -297,7 +315,6 @@ dl,
 table,
 figure {
     margin-top: 0;
-    // Spacing has been eyeballed to be 30px from the baseline.
     margin-bottom: unit(20px / @base-font-size-px, em);
```

Comment:
So it was always wrong for table and figure? Make sure you expand the view, because this includes

```
 p,
 ul,
 ol,
 dl,
```
as well